src/sentry/integrations/github/integration.py

```diff
@@ -267,29 +268,49 @@ def dispatch(self, request: Request, pipeline: Pipeline) -> Response:
             pipeline.bind_state("reinstall_id", request.GET["reinstall_id"])
 
         if "installation_id" in request.GET:
+            organization = self.get_active_organization(request)
+
+            # We want to wait until the scheduled deletions finish or else the post install to migrate repos do not work.
+            integration_pending_deletion_exists = OrganizationIntegration.objects.filter(
+                integration__provider=GitHubIntegrationProvider.key,
+                organization=organization,
+                status__in=[ObjectStatus.PENDING_DELETION],
```

Comment:
nit: If there's multiple statuses it makes sense to use `status__in` and have a const at the top of the file, but I think for this you're good with just `status=ObjectStatus.PENDING_DELETION`